builds/jenkins-build-pack-sls-app/Jenkinsfile

```diff
@@ -370,6 +366,7 @@ node() {
               }
             }
           }
+          createSubscriptionFilters(config, accountDetails, environment_logical_id, credsId)
```

Comment:
@raghits Do we need this? Aren't we using serverless plugin for this?

Reply:
Yes we are. This works for primary account but having issues with secondary accounts.

Reply:
Revert to use plugin.